test/samples/simple/input.json

```diff
@@ -1,13 +1,17 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"first_name": {
-			"type": "string"
-		},
-		"last_name": {
-			"type": "string"
-		}
-	}
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.